setup.py

```diff
@@ -37,7 +37,6 @@ def get_description():
         "tqdm",
```

Comment:
is tqdm really a requirement for map2loop? surely it can be optional?

Reply:
Yep, changed in https://github.com/Loop3D/map2loop/pull/109/commits/72aa32dff5c3ff3c0cdc4a987e92455bacb5b062